ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_mathematical_functions.py

```diff
@@ -2662,6 +2662,49 @@ def test_jax_power(
     )
 
 
+@handle_frontend_test(
+    fn_tree="jax.numpy.prod",
+    dtype_x_axis_dtype_where=_get_castable_dtypes_values(use_where=True),
+    keepdims=st.booleans(),
+    initial=st.one_of(st.floats(min_value=-100, max_value=100)),
+    promote_integers=st.booleans(),
+)
+def test_jax_prod(
+    dtype_x_axis_dtype_where,
+    keepdims,
+    initial,
+    promote_integers,
+    frontend,
+    backend_fw,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtypes, x, axis, dtype, where = dtype_x_axis_dtype_where
+    if ivy.current_backend_str() == "torch":
```

Comment:
Is the if statement present here necessary for some reason?

Reply:
Hi @bipinKrishnan - Sorry I mistakenly left it there as I was earlier trouble shooting a failed test. I have removed it now as it is certainly not required. Thanks.